src/telecom/voip/telecom-voip.constant.js

```diff
@@ -0,0 +1,47 @@
+export const TUC_TELEPHONY_ALIAS = {
+  featureTypes: {
```

Comment:
`conference` and `contactCenterSolution` seems not used, why do we need to add them ?

Split constants to avoid to have a huge constant, like `TELEPHONY_ALIAS_FEATURE_TYPES`, `TELEPHONY_ALIAS_SPECIAL_NUMBER` and so on.